2024-05-30 11:09:49,663 - apscheduler.scheduler - INFO - Scheduler started
2024-05-30 11:09:49,663 - telegram.ext.Application - INFO - Application started
2024-05-30 11:09:55,796 - __main__ - INFO - /start michaelizh 5090944878
2024-05-30 11:10:10,401 - __main__ - INFO - /sort michaelizh 5090944878
['1 6/11', ' 1.6'] по возрастанию
2024-05-30 11:10:25,982 - __main__ - INFO - /sort michaelizh 5090944878
['1 6/11', ' 1.61111'] по возрастанию
2024-05-30 12:24:56,422 - apscheduler.scheduler - INFO - Scheduler started
2024-05-30 12:24:56,423 - telegram.ext.Application - INFO - Application started
2024-05-30 12:24:59,692 - __main__ - INFO - /start michaelizh 5090944878
2024-05-30 12:25:28,007 - __main__ - INFO - /sort michaelizh 5090944878
['-1.541', ' 0.132', ' -0.532', ' 1.15', ' -1.15', ' 1 15/101', ' 1.611111111'] по возрастанию
2024-05-30 12:25:55,844 - __main__ - INFO - /start michaelizh 5090944878
2024-05-30 12:26:12,611 - __main__ - INFO - /calculations michaelizh 5090944878
['-1.541', '/', '-0.001']
2024-05-30 19:08:21,602 - apscheduler.scheduler - INFO - Scheduler started
2024-05-30 19:08:21,603 - telegram.ext.Application - INFO - Application started
2024-05-30 19:08:31,815 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2024-05-30 19:08:31,816 - apscheduler.scheduler - INFO - Scheduler has been shut down
2024-05-30 19:08:31,827 - telegram.ext.Application - INFO - Application.stop() complete
2024-05-30 19:08:33,344 - apscheduler.scheduler - INFO - Scheduler started
2024-05-30 19:08:33,344 - telegram.ext.Application - INFO - Application started
2024-05-30 19:08:35,186 - __main__ - INFO - /start michaelizh 5090944878
2024-05-30 19:08:58,778 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2024-05-30 19:08:58,778 - apscheduler.scheduler - INFO - Scheduler has been shut down
2024-05-30 19:08:58,790 - telegram.ext.Application - INFO - Application.stop() complete
2024-06-02 15:43:48,871 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:43:48,873 - telegram.ext.Application - INFO - Application started
2024-06-02 15:43:50,650 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:44:03,355 - __main__ - INFO - /calculations michaelizh 5090944878
['0.31', '/', '0.05']
2024-06-02 15:44:12,751 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:44:55,063 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:44:55,064 - telegram.ext.Application - INFO - Application started
2024-06-02 15:44:56,201 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:45:00,698 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:45:00,699 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_application.py", line 1264, in process_update
    await coroutine
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_handlers/basehandler.py", line 157, in handle_update
    return await self.callback(update, context)
  File "/Users/michael/Documents/Python_codes_all/bot/Calculator_copy.py", line 1139, in message_processing
    int(calc_numbers_lst[i + j][1]),
ValueError: invalid literal for int() with base 10: '05)'
2024-06-02 15:48:04,638 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:48:04,638 - telegram.ext.Application - INFO - Application started
2024-06-02 15:48:07,336 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:48:11,995 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:48:11,995 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_application.py", line 1264, in process_update
    await coroutine
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_handlers/basehandler.py", line 157, in handle_update
    return await self.callback(update, context)
  File "/Users/michael/Documents/Python_codes_all/bot/Calculator_copy.py", line 1141, in message_processing
    int(calc_numbers_lst[i + j][1]),
ValueError: invalid literal for int() with base 10: '05)'
2024-06-02 15:51:37,664 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:51:37,665 - telegram.ext.Application - INFO - Application started
2024-06-02 15:51:40,067 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:51:46,372 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:51:56,327 - __main__ - INFO - /calculations michaelizh 5090944878
['0.31', '/', '0.05']
2024-06-02 15:55:01,533 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:55:01,541 - telegram.ext.Application - INFO - Application started
2024-06-02 15:55:03,308 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:55:06,872 - __main__ - INFO - /calculations michaelizh 5090944878
['0.31', '/', '0.05']
2024-06-02 15:55:16,835 - __main__ - INFO - /calculations michaelizh 5090944878
['6', '1/5', '*', '1', '3/7']
2024-06-02 15:55:21,839 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:55:59,669 - apscheduler.scheduler - INFO - Scheduler started
2024-06-02 15:55:59,671 - telegram.ext.Application - INFO - Application started
2024-06-02 15:56:00,439 - __main__ - INFO - /start michaelizh 5090944878
2024-06-02 15:56:04,371 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '0.05)']
2024-06-02 15:56:12,884 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(-0.31', '/', '0.05)']
2024-06-02 15:56:19,792 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(0.31', '/', '-0.05)']
2024-06-02 15:56:26,334 - __main__ - INFO - /calculations michaelizh 5090944878
['1', '3/7', '*', '(-0.31', '/', '-0.05)']
2024-06-02 15:56:38,663 - __main__ - INFO - /calculations michaelizh 5090944878
['1.776124124', '*', '(0.31', '/', '0.05)']
2024-06-02 15:56:38,876 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_application.py", line 1264, in process_update
    await coroutine
  File "/Users/michael/Documents/Python_codes_all/bot/venv/lib/python3.9/site-packages/telegram/ext/_handlers/basehandler.py", line 157, in handle_update
    return await self.callback(update, context)
  File "/Users/michael/Documents/Python_codes_all/bot/Calculator_copy.py", line 1192, in message_processing
    raise IndexError
IndexError
2024-06-02 15:56:48,659 - __main__ - INFO - /calculations michaelizh 5090944878
['1.774', '*', '(0.31', '/', '0.05)']
